llvm/lib/Option/OptTable.cpp

```diff
@@ -33,25 +33,26 @@ using namespace llvm::opt;
 
 namespace {
 struct OptNameLess {
-  const char *StrTable;
-  ArrayRef<unsigned> PrefixesTable;
+  const StringTable *StrTable;
```

Comment:
Can this be a const reference instead? Is there a reason you chose to make this a pointer?

Reply:
(same as above)